src/components/Header/Header.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getUser } from '../../modules/user/reducer';
+import { store } from '../../index';
+import './style.css';
+
+
+class Header extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      userAuth: false,
+    };
+  }
+
+  userAuthObserver = () => {
+    if(this.props.user.isAuth){
+      this.setState({ userAuth: true });
+    }
+  }
+
+  componentDidMount = () =>{
+    store.subscribe(this.userAuthObserver);
+  }
+
+  foo = () =>{
+    console.log('work');
+  }
+
+
+  render() {
+    const { userAuth } = this.state;
+    return (
+      <header>
```

Comment:
`<header>` is not a thing you'd want to use in `body`. Replace it with h1 or some other heading tag. info about `header`: https://www.w3schools.com/tags/tag_head.asp
https://developer.mozilla.org/ru/docs/Learn/HTML/%D0%92%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5_%D0%B2_HTML/The_head_metadata_in_HTML

Reply:
I use tag <header>, not <head>.
http://htmlbook.ru/html/header

Reply:
I use tag <header>, not <head>. Maybe you mixed up? http://htmlbook.ru/html/header

Reply:
oh, sorry, you're right 😅